tests/functional/external_mode/test_virtualization_storageclass.py

```
@@ -0,0 +1,127 @@
+                )
+
+            # 2. Deploy VM Workload
+            log.info(f"Step 2: Creating VM '{vm_name}' using SC '{virt_sc_name}'.")
+            vm_obj.create_vm_workload(sc_name=virt_sc_name, ssh=False, verify=False)
+
+            # 2. Manually wait for ONLY the "Running" status
+            log.info(f"Waiting for VM {vm_name} to reach Running status...")
+            vm_obj.wait_for_vm_status(status=constants.VM_RUNNING, timeout=300)
+
+        except Exception as e:
+            log.error(f"Test logic failed: {str(e)}")
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)